README.md

```diff
@@ -309,6 +320,29 @@ This project contains a pre-configured configuration file `values.yaml`, you jus
 - `<your-container-registry>` - full-qualified hostname of your container registry
 - `domain`- full-qualified domain name used to access applications in your Kyma cluster
```

Comment:
`<your-release-name>` should also be added to this list. You can remove the note at the end related to this.

Reply:
I've made these changes.